src/Film.tsx

```diff
@@ -1,24 +1,13 @@
-import { FragmentType, useFragment } from "./gql/fragment-masking";
-import { graphql } from "../src/gql";
-
-export const FilmFragment = graphql(/* GraphQL */ `
-  fragment FilmItem on Film {
-    id
-    title
-    releaseDate
-    producers
-  }
-`);
+import { Film as GqlFilm } from "./gql/generated";
 
 const Film = (props: {
   /* `film` property has the correct type 🎉 */
-  film: FragmentType<typeof FilmFragment>;
+  film: GqlFilm;
 }) => {
-  const film = useFragment(FilmFragment, props.film);
```

Comment:
Instead of `useFragment`, there are a type `Film` which is auto generated.
In this case conflicting to the component name though